charts/zitadel/templates/debug_replicaset.yaml

```diff
@@ -17,7 +17,6 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/configmap: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
Like eliobischof commented, configmap.yaml was renamed to configmap_zitadel.yaml in 9.x:
checksum/configmap: {{ include (print $.Template.BasePath "/configmap_zitadel.yaml") . | sha256sum }}